packages/protocol/contracts/L1/TaikoToken.sol

```diff
@@ -51,8 +51,8 @@ contract TaikoToken is EssentialContract, ERC20SnapshotUpgradeable, ERC20VotesUp
     }
 
     /// @notice Creates a new token snapshot.
-    function snapshot() public onlyFromOwnerOrNamed("snapshooter") {
-        _snapshot();
+    function snapshot() public onlyFromOwnerOrNamed("snapshooter") returns (uint256) {
```

Comment:
BridgedERC20 also has a `snapshot` function that you need to change. If this is a bug, then we should have a fix(protocol)... PR.

Reply:
See https://github.com/taikoxyz/taiko-mono/pull/16436